### ISSUE TITLE
stateful models

### ISSUE THREAD

**Issue by whzghb** [triton-inference-server/server#7360]
When tritonserver is deployed as multiple pods, for stateful models, does traffic have to be sent to the pod of the last inference request? If so, how is it done?

**Comment by statiraju** [comment]
@Tabrizian can you help answer here.

**Comment by Tabrizian** [comment]
@whzghb Yes, when deploying stateful models you need to make sure to send all the request with the same correlation ID to the same Triton instance.

**Comment by whzghb** [comment]
> @whzghb Yes, when deploying stateful models you need to make sure to send all the request with the same correlation ID to the same Triton instance.

That means it depends on the client, not the server, right?

**Comment by fapont** [comment]
Are there any news on this subject @Tabrizian ? For instance a multi instance support like that implemented with [tensorrt-llm backend](https://github.com/triton-inference-server/tensorrtllm_backend?tab=readme-ov-file#leader-mode).

This could drastically ease the deployment of such models at scale on Kubernetes.
